SofaKernel/modules/Sofa.Testing/tools/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(Sofa.Testing.Tools)
+
+set(SOFATESTINGTOOLS_SRC "src/sofa/testing/tools")
+
+# only header files
+set(HEADER_FILES
+    ${SOFATESTINGTOOLS_SRC}/MassSpringSystemCreation.h
+    ${SOFATESTINGTOOLS_SRC}/SunPlanetSystemCreation.h
+)
+
+set(SOURCE_FILES
+    ${SOFATESTINGTOOLS_SRC}/empty.cpp # because of not interface type
```

Comment:
What's the reason this project is not an INTERFACE?

Reply:
SOFA macros does not support the INTERFACE keyword